Sources/MQ/Errors/Undefined.swift

```diff
@@ -9,13 +9,16 @@ public struct Undefined: TheError {
 	/// - Parameters:
 	///   - message: Message associated with this error.
 	///   Default value is "Undefined".
+	///   - underlyingError: Underlying, unrecognized error if any.
+	///   Default value is `.none`.
 	///   - file: Source code file identifier.
 	///   Filled automatically based on compile time constants.
 	///   - line: Line in given source code file.
 	///   Filled automatically based on compile time constants.
 	/// - Returns: New instance of ``Undefined`` error with given context.
 	public static func error(
 		message: StaticString = "Undefined",
+		underlyingError: Error? = .none,
```

Comment:
I'm wondering if it's makes sense to define underlying error in this case. "Undefined" error suggests that it's some kind of error that we have not foreseen - isn't that what "TheError" is for in the first place? Shouldn't undelyingError be implemented in TheError?

Reply:
Yeah, You might be right, I will add the new error for that case i.e. "Unhandled" to have an option with fallback form any `Error` to `TheError` without defining concrete one